core/controller/Dockerfile

```diff
@@ -8,6 +8,21 @@ ENV UID=1001 \
 ENV SWAGGER_UI_DOWNLOAD_SHA256=3d7ef5ddc59e10f132fe99771498f0f1ba7a2cbfb9585f9863d4191a574c96e7 \
     SWAGGER_UI_VERSION=3.6.0
 
+ENV DOCKER_VERSION=1.12.0 \
```

Comment:
please comment this whole section that it's needed for lean mode where the controller is also an invoker.